src/modules/projects/components/ExternalSubmissionsViewModal.tsx

```diff
@@ -134,17 +134,19 @@ const ExternalSubmissionsModalContent = ({
               minRows={2}
               onChange={(event) => setNotes(event.target.value)}
             />
-            <LabeledCheckbox
-              value={status === ExternalFormSubmissionStatus.Reviewed}
-              label='Mark as Reviewed'
-              onChange={(_e, checked) => {
-                setStatus(
-                  checked
-                    ? ExternalFormSubmissionStatus.Reviewed
-                    : ExternalFormSubmissionStatus.New
-                );
-              }}
-            />
+            {submission.status === ExternalFormSubmissionStatus.New && (
+              <LabeledCheckbox
+                value={status === ExternalFormSubmissionStatus.Reviewed}
+                label='Mark as Reviewed'
+                onChange={(_e, checked) => {
+                  setStatus(
+                    checked
+                      ? ExternalFormSubmissionStatus.Reviewed
+                      : ExternalFormSubmissionStatus.New
+                  );
+                }}
+              />
+            )}
             <LabeledCheckbox
               value={spam}
               label='Mark as Spam'
```

Comment:
I wonder if we should also disable 'Mark as Spam' when the submission has already been reviewed. It seems meaningless to mark a submission as spam at that point, if the cdes/client/enrollment have already been created and we aren't taking any action to delete them. All it does is hide the submission from the default view of this table.

Reply:
yeah I agree, we could hide it